script/github_pr_errors

```diff
@@ -344,6 +349,23 @@ class JobErrorsFinder
       end
   end
 
+  def find_failures_explanation(log)
+    explanations = []
+    log.split("\n").each do |line|
+      if line.end_with?("Failures:") .. line.end_with?("Failed examples:")
```

Comment:
What is this called? 😅 Never seen this before

Reply:
flip flop operator. I discovered it while looking for a way to search for a range in a list. Maybe I should add a comment about it.